### PR TITLE
upgrade extensions' version (Keplr, MM)

### PR DESCRIPTION
**Pull request by pgoos**
This PR upgrades Keplr (0.8.1 -> 0.8.9) and MM (9.5.9 -> 9.6.1).
_Important note for future updates_: an extension needs to be **first installed in Chrome**/chromium and then containing folder can be zipped and put into the repo.

### CODE REVIEW

File: ui/e2e/config.js [hunk 1]
```diff
@@ -17,10 +17,10 @@ export const KEPLR_CONFIG = {
 };
 
 export const MM_CONFIG = {
-  id: "nkbihfbeogaeaoehlefnkodbefgpgknn",
-  ver: "9.1.1_0",
+  fileId: "kbncgnhjhkdmacanlpmplcbbncoblkfo", // zip name id is static, where ext id is dynamic and now stored in jest globals config
```
Comment (ryardley) [inline ui/e2e/config.js:20]: This looks odd the id should not change. Can we get on a call about this? Are you sure you have used the correct zips from https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn

Comment (pgoos) [reply ui/e2e/config.js]: You were right @ryardley , changing id was unexpected. I updated PR description (and the whole PR) that describes what needs to be done to have non-changing id.

File: ui/e2e/utils.js [hunk 2]
```diff
@@ -39,6 +39,32 @@ export async function extractExtensionPackage(extensionId) {
   return;
 }
 
+// not used right now. It's not necessary to use it when id is static
```
[the remark this answers is not in the document]
Comment (pgoos) [reply ui/e2e/utils.js:42]: Hmm how about keeping the method if we want to extract the id from url for some reason in the future? Would keeping the method be confusing for you?

[the remark this answers is not in the document]
Comment (ryardley) [reply ui/e2e/utils.js:42]: But it will always be a static id?

[the remark this answers is not in the document]
Comment (pgoos) [reply ui/e2e/utils.js:42]: Yeah, we expect it to be static.

**Review by ryardley — APPROVED** [review]

**Pull request merged**